index.html

```diff
@@ -370,9 +370,9 @@
     <div class="modal-dialog modal-lg">
       <div class="modal-content">
         <div class="well">
-          <h3>Excerpt from beginning</h3>
+          <h3>Trim</h3>
           <p><code>ffmpeg -i <i>input_file</i> -t <i>5</i> -c copy <i>output_file</i></code></p>
-          <p>This command captures a certain portion of a video file, starting from the beginning and continuing for the amount of time (in seconds) specified in the script. This can be used to create a preview file, or to remove unwanted content from the end of the file. To be more specific, use timecode, such as 00:00:05.</p>
+          <p>This command trims the beginning and end of a video file without re-encoding</p>
```

Comment:
If I remember carefully, the idea of this receipt was e.g. to capture the opening titles for cataloguing purposes.